scripts/test_screeps_rl_live_dashboard.py

```
@@ -1866,6 +1901,69 @@ def test_successful_refresh_does_not_cache_in_progress_summary(self) -> None:
         self.assertFalse(first["refresh"]["refreshInProgress"])
         self.assertFalse(second["refresh"]["refreshInProgress"])
 
+    def test_successful_refresh_keeps_primed_summary_cache_after_finishing(self) -> None:
+        original_refresh_metrics = live.refresh_metrics
+        original_build_live_summary = live.build_live_summary
+        build_calls: list[JsonObject] = []
+
+        def successful_refresh_metrics(
+            db_path: Path,
```

Ruff (0.15.14)

[warning] Unused function argument: `db_path`

(ARG001)